### ISSUE TITLE
Unable to edit redirect_uri & redirect_uri

### ISSUE THREAD

**Issue by lucien144** [Kdyby/Facebook#45]
Hey,
Is there any reason why there's no possibility of changing `redirect_uri` & `redirect_uri` params to something other than `$this->currentUrl` ?

May this be possible in any future update? (I'm keen to prepare a pull request with this)

Thanks.

**Comment by fprochazka** [comment]
Since the returnUrl should point to the signal of the component it doesn't make sense to me to allow it.
But if you give me a good use-case. I'm not strictly against changing it.

**Comment by lucien144** [comment]
Well,
if you want to use FB login within FB tab (~ app in Facebook Page), it might be handy to ended up on different url from where you can redirect user back to FB tab.

You can do this on the initial page of course, but it might lead to redirect loop within the FB tab.

**Comment by fprochazka** [comment]
@lucien144 that's what the signal query param is for, no?

**Comment by lucien144** [comment]
@fprochazka Yeah, seems like this should be enough what you need. Thanks for pushing me to the right direction!

**Comment by fprochazka** [comment]
@lucien144 let me know afterwards if it worked for you :)

**Comment by lucien144** [comment]
@fprochazka Sorry, I thought I get it but probably no :) I've thought you are talking about signal params, eg. if I do this `<a n:href="fbLogin-open!, isFbTab => 1">Login with Facebook</a>` then I don't know where to get the `$isFbTab` on the response. Can you show me what did you mean?

Thank you.
